genie-docs/src/docs/asciidoc/concepts/_netflixExample.adoc

```diff
@@ -65,6 +65,7 @@ downloaded into the job working directory at runtime.
     "s3://bucket/users/bdp/h2prod/20161217/205111/genie/hdfs-site.xml",
     "s3://bucket/users/bdp/h2prod/20161217/205111/genie/core-site.xml"
   ],
+  "dependencies": [],
```

Comment:
We may want to update these examples with our "new" way of doing things with HoS